docs/guides/DolphinPay.md

```diff
@@ -1,268 +1,261 @@
 
-        1.1 click on polkadot.js extension > Click on three vertical dots as mentioned in the image > Select Calamari Parachain.
+3. How can I recover private tokens if I forgot my Manta Signer password?
 
-        If you still don't see the "Calamari" parachain option, then please visit this URL https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fcalamari-rpc.dwellir.com#/explorer.
+   See our [guide to Manta Signer](https://docs.manta.network/docs/guides/MantaSigner) for more details.
 
-   2. Go to settings > Metadata > Update Metadata
+4. How long do trnsactions take? Why is my transaction taking so long?
 
-      Then please follow steps mentioned in 1.1
+  After you have approved a transaction with your browser wallet, it should take about 45 seconds to finalize. Occasionally, transactions with many small inputs require multiple "batches"; you will have to to approve the first part of the transaction, wait for the first part of the transaction to finalize, and then approve the next part.
 
-6. The `To Private` button on the dolphin testnet is not working
+  If your transaction is taking a long time, make sure Manta Signer and your browser wallet aren't awaiting approvals; you might not have sent actually the transaction. Otherwise, if your transaction keeps you waiting for more than a few minutes, please [file a bug report.](https://discord.gg/c72QMWEVyY)
 
-   Please try reloading the page at https://app.manta.network/dolphin/transact. Click on the "To Private" button again once you see a number next to BOTH "Balance" fields. Wait patiently until the three dots in the private section `...` disappear.
+5. How does private payment work?
 
-   Please note that our testnet is currently overwhelmingly popular, and many transactions are trying to go through the network simultaneously, so if you're still experiencing.
+   [Find out more here.](../learn/PrivatePayment.md)
 
-   1. Public to private tx not activating
-   2. Polkadot.js wallet not showing
-   3. Long periods for transactions
+6. How do I find my Calamari address on polkadot.js?
 
-   Please wait and try again later!
+   Click on polkadot.js extension, then click on three vertical dots next to your address, and then select "Calamari Parachain." If you don't see the "Calamari Parachain" option, update your extensions metadata [here](https://polkadot.js.org/apps/#/settings/metadata).
 
-7. To remove your private account data completely and set up a new account, you should remove these files:
+7. How can I delete my Manta Signer Account?
 
-      - macOS: `~/Library/Application Support/manta-signer/`
-      - Linux: `~/.config/manta-signer/`
-      - Windows: `~/AppData/Roaming/manta-signer/`
+   See our [guide to Manta Signer](https://docs.manta.network/docs/guides/MantaSigner) for more details.
 
 8. How long will the testnet run?
 
     The testnet will run forever 👾
```

Comment:
i don't think this is accurate. I suggest
" We will run various iterations of testnets over time and reset the network (after announcement) when necessary. Do not expect your testnet assets to be available forever.